indes.html

```diff
@@ -12,39 +12,31 @@
 <body>
     <div class="container">
         <div class="calc">
-            <div class="calc_content">
-                <div class="calc_header">
-                    <h1>SIMPLE CALCULATOR</h1>
+            <div class="calc__content">
+                <div class="calc__header">
+                    Simple calculator
                 </div>
-                <div class="calc_box">
-                    Number 1
-                    <div class="calc_number"> 
-                        <input type="number">
-                    </div>
+                <div class="calc__box">
+                    <label for="number 1">Number 1</label>
```

Comment:
@lucazmartins  O valor do atributo `for` deve ser sempre igual ao valor do `id` do input. Então nesse caso seria: 
```html
for="num1"
```